dearpygui_obj/node.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+from typing import Tuple, List
+
+from dearpygui import core as dpgcore
+from dearpygui_obj import _register_item_type
+from dearpygui_obj.wrapper import PyGuiWidget
```

Comment:
Needs to be updated to reflect new placement of PyGuiWidget

from dearpygui_obj.wrapper.widget import PyGuiWidget

Reply:
👍